### PR TITLE
Added lastElement code

### PR DESCRIPTION
**Pull request by ritikkuril**
done

### CODE REVIEW

File: .metals/metals.lock.db [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #FileLock
-#Wed Mar 15 17:50:15 IST 2023
-server=localhost\:62132
```
Comment (tg-trainer) [inline .metals/metals.lock.db:3]: don't commit these extra files

Comment (tg-trainer) [reply .metals/metals.lock.db:3]: also, resolve merging conflicts

**Comment by ritikkuril** [comment]
fixed the conflict @tg-trainer

**Comment by ritikkuril** [comment]
i think code is ready for pull @tg-trainer

File: src/main/scala/RitikKuril/DropNelement.scala [hunk 2]
```diff
@@ -0,0 +1,21 @@
+package RitikKuril
+
+object DropNelement{
+
+  def drop(list:List[Int],n:Int):List[Int]={
+
+  //  list.patch(n-1,Nil,1); (in build method)
+
+   val(x,y)= list.splitAt(n-1);
```
Comment (tg-trainer) [inline src/main/scala/RitikKuril/DropNelement.scala:9]: try to use the recursion

File: src/main/scala/RitikKuril/LastElementList.scala [hunk 3]
```diff
@@ -0,0 +1,15 @@
+package RitikKuril
+
+object LastElementList {
+    def Last(list:List[Int]):Int={
```
Comment (tg-trainer) [inline src/main/scala/RitikKuril/LastElementList.scala:4]: convention

**Pull request closed without merging**